### ISSUE TITLE
Accessibility actions in messages and chat list are missing

### ISSUE THREAD

**Issue by Jonathans859** [signalapp/Signal-Android#14206]
### Guidelines

- [x] I have searched searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

Accessibility actions are a common practice for blind people like me to interact with apps via their Screen Readers, not a feature, an essential thing which misses and would make the app way more usable for many people. Please see Google's developer docs below to understand what actions do and how to fix that problem. Essentially they allow people who rely on a screenreader to interact with functions of the app in a much quicker way. Thanks for consideration.
https://developer.android.com/guide/topics/ui/accessibility/principles#accessibility-actions

### Device

Google Pixel 9 Pro

### Android version

16

### Signal version

v7.45.2

**Comment by jonathans859** [comment]
To clarify, those actions would include features to be more easily reachable which are now taking longer to access through the long pressing of a message, muting a chat or deleting a message, for instance, all the features from the context menus.

**Comment by jonathans859** [comment]
So, anything new on this?

**Comment by jonathans859** [comment]
Greetings, interest in accessibility, anyone?

**Comment by s4p3r-c1n0s** [comment]
I will take this one, and make sure talk back (or any other form of a11y) is able to run through the long press menu items for chats and messages.

**Comment by jonathans859** [comment]
> I will take this one, and make sure talk back (or any other form of a11y) is able to run through the long press menu items for chats and messages.

Very much appreciated. Can't wait to see what you come up with.

**Comment by s4p3r-c1n0s** [comment]
Most of it is taken care of, at this moment message reactions that pop up for a message is not yet exposed to a11y. Can keep a separate ticket for it.

Looking for PR reviews

**Comment by s4p3r-c1n0s** [comment]
creating separate PRs for messages screen a11y and chat list a11y
